website/docs/artist_tools_tray.md

```diff
@@ -0,0 +1,71 @@
+---
+id: artist_tools_tray
+title: Tray
+sidebar_label: Tray
+description: OpenPype general menu.
+---
+# Tray
+
+Click on the **OpenPype** logo to open the tray and find the major informations about your account and especially your studio files. You can customize your tray by activating options in the **Studio Settings** such ass adding **Ftrack** or **Kitsu**.
+
+![OpenPype Tray](assets/tools/tools_openpype_tray.png)
+
+## Ftrack
+*Enable it in the [Studio Settings](admin_settings_system.md).*
+### Credentials
+The **Credentials** contains the studio's **Ftrack URL**, your **Username** and the **API Key** given by your studio. 
```

Comment:
more information ![here](artist_ftrack.md#login-to-ftrack-module-in-openpype-best-case-scenario)